pages/made-in-youngstown.tsx

```diff
@@ -8,13 +8,13 @@ export default function MadeInYoungstown() {
   return (
     <Layout page="Made In Youngstown" description={pageDescription}>
       <h1 className="text-3xl p-2">Made In Youngstown</h1>
-      <p className="sm:text-2xl text-xl p-2">
+      <p className="sm:text-xl text-md p-2">
```

Comment:
Let's set the text that's in this `p` to `pageDescription` and use that var here.